internal/api/xml/player.go

```diff
@@ -0,0 +1,8 @@
+package xml
+
+// Player represents player info
```

Comment:
**[BETTER-UX]:** I would like to add an example of each XML structure in the form of a comment (like doc-string), so it might be easier for 3r party developers to understand what to expect in XML. It does not mean, it has to be done now. Consider it just a nice-to-have.